use/spatial-stat_SelectionSNP.Rmd

```diff
@@ -0,0 +1,231 @@
+---
+title: "Detection of the Signal of selection from genome scan. Population based analysis "
+---
+
+# Introduction
+
+The purpose of the vignette is to detect outlier SNP loci from a large number of SNPs genotyped for a large number of individuals. Individuals are organised in well defined populations.  Then, we  tested if allele counts in outlier loci are correlated to the temperature.   We will first use a method based on principal component analysis (PCADAPT) (section 2) and a second method based on atypical values of Fst (OUTFLANK; Whitlok and Lotterhos (2015) ; https://github.com/whitlock/OutFLANK) (section 3) to detect outliers loci. Then,  we will test if the allele counts in outliers are correlated to the temperature using a poisson regression (section 4).  
+
+ 
+
+# Assumptions
+
+- We assume that loci are neutral and we are looking for those are deviant from the neutral assumption.
+- We assume that loci correlated to the environment (temperature) are under selection. 
+
+# Data
+
+We will use a simulated data provided by Katie Lotterhos (Lotterhos et al. in prep) 
+THe initial data set is constituted of 509 individuals, 19 populations and 3084 SNP. 
+2 extra-columns: population labels, temperature
+SNP are in columns: each row contains one value at each SNP loci (separated by spaces or tabulations) corresponding to the number of reference alleles: 0 means zero copies of the reference allele, 1 means one copy of the reference allele (i.e. heterozygote), and 2 means two copies of the reference allele.   
+
+
+# Resources/Packages required
+
+
+PCADAPT:http://membres-timc.imag.fr/Michael.Blum/PCAdapt.html
+OutFlank:https://github.com/whitlock/OutFLANK
+
+
+Loading the required packages:  
+

```

Comment:
For the required packages, the process for installation is shown, but not all of the packages are loaded, which will cause errors for users following along. My suggestion: have one unevaluated showing how to install OutFLANK (as it is described in the outflank readme) and then load the packages in a visible chunk

    Since OutFLANK is not yet on CRAN, you will need to install it from GitHub using the *devtools* R package.
    Additionally, OutFLANK depends on *qvalue* from [Bioconductor](http://www.bioconductor.org/packages/release/bioc/html/qvalue.html), 
    which must be installed beforehand.

    ```{r, packages, eval=FALSE}
    library("devtools")
    source("https://bioconductor.org/biocLite.R")
    biocLite("qvalue")
    install_github("whitlock/OutFLANK")
    ```
    
    Loading the required packages:

    ```{r, loadpackage, message = FALSE}
    library(pcadapt)
    library(qvalue)
    library(OutFLANK)
    ```